grr/server/grr_response_server/gui/ui/components/human_readable_size/human_readable_size.ng.html

```diff
@@ -0,0 +1 @@
+<span>{{getHumanSize()}}</span>
```

Comment:
(optional) Existing code that renders byte values (check out the form when selecting the flow "Collect single file" which comes from grr/server/grr_response_server/gui/ui/components/flow_args_form/collect_single_file_form.ng.html) links the unit to Wikipedia. You don't need to follow the same pattern, but I thought it's cool to clarify what `MiB` and similar mean.